js/src/sdk/utils/telemetry/index.ts

```diff
@@ -32,12 +32,15 @@ export class TELEMETRY_LOGGER {
   private static async sendTelemetry(payload: Record<string, unknown>[]) {
     const isTelementryDisabled =
       getEnvVariable("TELEMETRY_DISABLED", "false") === "true";
+
     if (isTelementryDisabled) {
       return;
     }
+
     const url = `${TELEMETRY_URL}/api/sdk_metrics/telemetry`;
+
     const reqPayload = {
-      data: { events: payload },
+      data: payload,
```

Comment:
The change in payload structure from `{ data: { events: payload } }` to `{ data: payload }` is a breaking change. Make sure this is compatible with the backend API and consider versioning the API if needed.